docs/examples/extras.md

```diff
@@ -0,0 +1,75 @@
+# Extras
+
+## Save Config Files
+
+This example lets you save the parameters of the invoked command to a configuration file using the `@save_config` decorator which operates on Typer commands (requested in [issue #25](https://github.com/maxb2/typer-config/issues/25)).
+
+An example typer app:
+```{.python title="simple_app.py" test="true"}
+import typer
+from typer_config import json_conf_callback
+from typer_config.extras.commands import save_config
+from typer_config.extras.dumpers import json_dumper
+
+
+app = typer.Typer()
+
+@app.command()
+@save_config(json_dumper, "./saved.json") # MUST BE AFTER app.command() (2)
```

Comment:
Could we potentially add json_dumper as a default variable here? (To further simplify the API)

Reply:
I will make `@save_json_config` to match the rest of the API (`@use_json_config`, `json_conf_callback`, etc.). I don't want to choose a default file format because everyone will have their personal favorite :sweat_smile: